models/Drone.model.js

```diff
@@ -1 +1,12 @@
-// Iteration #1
\ No newline at end of file
+// Iteration #1
+const { Schema, model } = require("mongoose");
+
+const droneSchema = new Schema ({
+    name: String,
+    propellers: Number,
+    rmaxSpeed: Number,
```

Comment:
Be careful with the model. You passed `rmaxSpeed` here, and then on the routes, you passed `speed`, and on the views, `maxSpeed`. Remember that they should always be the same. The correct version here and eveyrwhere else would be `maxSpeed`.